infra/scripts/version-debug-linker-flags.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+echo " \
```

Comment:
Given that this line is the only way this file differs from `infra/scripts/version-linker-flags.sh`, can we have that file call this one? Or just have a single script that takes a debug argument and contains a conditional?

I'd like to avoid duplicating code here.